hw/ip_templates/clkmgr/dv/env/seq_lib/clkmgr_base_vseq.sv.tpl

```diff
@@ -206,8 +206,8 @@ ${spc}ral.${src}_meas_ctrl_shadowed.lo};
     csr_wr(.ptr(meas_ctrl_regs[which].en), .value(MuBi4False));
   endtask
 
-  local function int get_meas_ctrl_value(int min_threshold, int max_threshold, uvm_reg_field lo,
-                                         uvm_reg_field hi);
+  function int get_meas_ctrl_value(int min_threshold, int max_threshold, uvm_reg_field lo,
```

Comment:
Presumably we're losing the `local` because we want to call it in `clkmgr_regwen_vseq`. Would `protected` work? (Clearly not a big deal but I wondered about it)

Reply:
Okay: at some point I changed it to `protected` but changed my mind since it seemed unnecessary. You changed the balance :smiley:

Done